### DISCUSSION TITLE
Boosting Godot 3D's Performance and Editor Stability for Better Game Development

### DISCUSSION THREAD

**Discussion by OURABIG** [godotengine/godot-proposals#7957] [General Discussions]
I'm a big fan of Godot and its creative approach to game making. But there are some big problems in Godot 3D that are holding it back from being great for making games that run smoothly. While Godot has lots of cool features, I think we should stop adding new stuff for a bit and focus on making it work better first.

Performance Problems:

Games Slow Down: Godot's 3D games often get slow, especially when there are lots of things happening on the screen. This makes it hard to create games that feel good to play.

Uses Too Much Memory: Godot 3D games sometimes use more computer memory than they should, which can make them crash or run poorly, especially on older computers.

Editor Stability:

Crashes: The Godot 3D editor can sometimes crash, causing frustration and work interruptions for developers. Improving editor stability will make the game development process smoother.
Physics Problems:

Physics Can Be Weird: Sometimes, the physics in Godot 3D don't work quite right, especially when things are supposed to bump into each other or move fast. This can make games feel unpredictable and frustrating.

Problems with Objects: Games with lots of things that move around can get really slow because the physics system has trouble keeping up.

Why Performance Matters:
Fixing these performance, editor stability, and physics problems is important because it will make Godot 3D better for making games. When games run smoothly and predictably, players have more fun, and that's what we want.

**Comment by Zireael07** [comment]
This wishlist list is not actionable.
